securedrop/tests/functional/functional_test.py

```diff
@@ -11,6 +11,8 @@
 import sys
 
 
+# Set environment variable so config.py uses a test environment
+os.environ['SECUREDROP_ENV'] = 'test'
```

Comment:
I see why you're moving this here, but remove the old setting of the environmental variable on line 29